extensions/flyway/deployment/src/main/resources/dev-ui/qwc-flyway-datasources.js

```diff
@@ -90,7 +91,16 @@ export class QwcFlywayDatasources extends QwcHotReloadElement {
                 </vaadin-button>`;
         }
     }
-    
+
+    _renderUpdateButton(ds) {
+        if(ds.hasMigrations){
+            return html`
+                <vaadin-button theme="small" @click=${() => this._showCreateDialog(ds)} class="button" title="Create update migration file. Always manually review the created file as it can cause data loss">
```

Comment:
I think you need to add a different dialog, because the one you're opening is designed for the "create" button and will just create the initial migration file, not the update one.

Reply:
this is now fixed